servers/physics_3d/area_3d_sw.cpp

```diff
@@ -163,6 +163,20 @@ void Area3DSW::set_param(PhysicsServer3D::AreaParameter p_param, const Variant &
 		case PhysicsServer3D::AREA_PARAM_PRIORITY:
 			priority = p_value;
 			break;
+		case PhysicsServer3D::AREA_PARAM_WIND_FORCE_MAGNITUDE:
+			wind_force_magnitude = p_value;
+			ERR_FAIL_COND_MSG(wind_force_magnitude >= 0, "Wind force magnitude must be a non-negative real number, but a negative number was specified.");
```

Comment:
The error condition should be `wind_force_magnitude < 0`, and it should be before setting the value, so it can return without setting the wrong value when it fails.

Reply:
LOL man I must've been pretty tired at this point. Thanks for catching these.

Reply:
No problem :) The error is still after setting the value in the latest version, it should be before.